README.md

```diff
@@ -32,6 +32,7 @@ gulp.task('test', function (cb) {
       gulp.src(['test/*.js'])
         .pipe(mocha())
         .pipe(istanbul.writeReports()) // Creating the reports after tests runned
+        .pipe(istanbul.enforceThresholds({ thresholds: { global: 90 } })) // Enforce a coverage of at least 90%
```

Comment:
no need to nest this deeper than necessary. I don't think the `threshold` key is necessary.